src/sentry/integrations/vsts/integration.py

```diff
@@ -188,6 +185,18 @@ def build_integration(self, state):
             },
         }
 
+        try:
+            IntegrationModel.objects.get(provider='vsts', external_id=account['AccountId'])
```

Comment:
So is the idea that if there's an `Integration` that matches here, we must have already setup the subscription? Is that always the case – is there a way we could have the object, but not have created the subscription?

Reply:
No. There shouldn't be. It's part of the creation process. But I can add a check to verify that the subscription is there as well.